source/new-starter/access/index.html.md.erb

```diff
@@ -1,10 +1,63 @@
 ---
-title: Access
+title: Get access to required apps
 weight: 23
 last_reviewed_on: 2020-12-10
 review_in: 6 months
 ---
 
-# Access
+# Get access to required apps and information
 
-List of all the things you should have access to and how to get that access
+When you join the GOV.UK Account team, you must get access to all the required apps and information you need.
+
+## Complete the GOV.UK new starter process
+
+If you are new to GDS or the GOV.UK programme, you must complete the GOV.UK new starter process.
```

Comment:
Should we make it clear that only tech people are required to do these tasks?

Reply:
Given that the intended user of the team manual is developers, and this is stated on the manual landing page, i think we're ok.

Reply:
But ok i'll make it clear again